### PR TITLE
fix(HACBS-1262): use mapped location in create-pyxis-image task

### PR DESCRIPTION
**Pull request by johnbieren**
This commit switches the create-pyxis-image task to use the mapped location in the snapshot (repository) instead of the origin location (containerImage).

Signed-off-by: Johnny Bieren <jbieren@redhat.com>

### CODE REVIEW

File: catalog/task/create-pyxis-image/0.1/create-pyxis-image.yaml [hunk 1]
```diff
@@ -69,7 +69,7 @@ spec:
         echo "${pyxisCert}" > /tmp/crt
         echo "${pyxisKey}" > /tmp/key
 
-        for containerImage in $(jq -r '.components[].containerImage' <<< '$(params.snapshot)') ; do
+        for containerImage in $(jq -r '.components[].repository' <<< '$(params.snapshot)') ; do
```
Comment (scoheb) [inline catalog/task/create-pyxis-image/0.1/create-pyxis-image.yaml:72]: this is the entry in the json that has the `external` pullspec?

Comment (johnbieren) [reply catalog/task/create-pyxis-image/0.1/create-pyxis-image.yaml:72]: Yes, the apply-mapping task turns a snapshot like this
```
{"application":"simple-python","components":[{"name":"devfile-sample-python-basic","containerImage":"quay.io/jbieren/user-workload@sha256:3b38e726ff15eb20a3bfbe0cfa831cdcf68305e2ddbbf437b5204a51cc6b4b6d"}],"artifacts":{}}
```
into this
```
{"application":"simple-python","components":[{"name":"devfile-sample-python-basic","containerImage":"quay.io/jbieren/user-workload@sha256:3b38e726ff15eb20a3bfbe0cfa831cdcf68305e2ddbbf437b5204a51cc6b4b6d","repository":"quay.io/jbieren/shebert-dc-metro-map"}],"artifacts":{}}
```

Comment (scoheb) [reply catalog/task/create-pyxis-image/0.1/create-pyxis-image.yaml:72]: but that entry does not have a sha or tag?

Comment (johnbieren) [reply catalog/task/create-pyxis-image/0.1/create-pyxis-image.yaml:72]: you'd know better than me. It is whatever the entry is in the mapping file. I don't see a sha or tag in the example `strategy-config` repos I know of. `skopeo inspect` still works on it though, as it is just using `latest` by default (same as the push we do afaik)

**Review by scoheb — APPROVED** [review]

**Pull request merged**